concepticondata/conceptlists/List-2014-1280.tsv

```diff
@@ -517,7 +517,7 @@ List-2014-1280-515	1623	SPEAK	515	speak, talk	tell story	20	http://clics.lingpy.
 List-2014-1280-516	1866	STUTTER	516	stutter, stammer	stutter, stammer	238	http://clics.lingpy.org/browse.php?community=cluster_238_stutter, stammer	0	http://clics.lingpy.org/browse.php?community=network_stutter, stammer_0&view=part	227	11	11	1161
 List-2014-1280-517	1458	SAY	517	say	tell story	20	http://clics.lingpy.org/browse.php?community=cluster_20_tell story	27	http://clics.lingpy.org/browse.php?community=network_say_27&view=part	308	73	129	37
 List-2014-1280-518	1711	TELL	518	tell story	tell story	20	http://clics.lingpy.org/browse.php?community=cluster_20_tell story	28	http://clics.lingpy.org/browse.php?community=network_tell story_28&view=part	251	44	97	124
-List-2014-1280-519	1115	SPEECH	519	speech (make a)	tell story	20	http://clics.lingpy.org/browse.php?community=cluster_20_tell story	19	http://clics.lingpy.org/browse.php?community=network_speech (make a)_19&view=part	218	43	93	142
+List-2014-1280-519	3209	MAKE A SPEECH	519	speech (make a)	tell story	20	http://clics.lingpy.org/browse.php?community=cluster_20_tell story	19	http://clics.lingpy.org/browse.php?community=network_speech (make a)_19&view=part	218	43	93	142
```

Comment:
did you check the reflexes? I mean: what does KEY have? You know, the "(make a)" can well be a gloss exemplifying what the noun is, it is crucial to check with KEY and WOLD, to make sure what was intended if you want to change it.

Reply:
WOLD has "the speech" (and is linked to the noun) but Key has "speech (make a) [english]; discours (faire une) [french]; discursar  [portuguese]; речь (сделать) [russian]; pronunciar un discurso [spanish]", i.e. it rather seems to refer to the verb, so I left it there. On IDS however, both meanings can be found depending on the language: https://ids.clld.org/parameters/18-222#2/20.3/149.6

Reply:
Nope, it is entirely not clear where it refers to to me. It is just a very messy concept list we have there. Only portugues has a verb in the gloss, all others have a noun and a verb in brackets. Please link to speech for now. The link is as arbitrary, and we want to keep the links between IDS and WOLD, and key is just ids.

Reply:
So, Key, List-2014 and Haspelmath should always be linked to the same concept? In case of clear differences, on which one should I base my choice? Haspelmath is the only one with further information in the form of example sentences of these 3, as far as I see, and the oldest one.

Reply:
Look, List-2014 is DIRECTLY taken from IDS, and IDS is Key-2016. This is also mentioned in the note, at least the overlap of 1280 concepts is both reflecting IDS and WOLD. IDS is only in translated form, so you COULD always check them. WOLD and IDS also both have their internal IDs, that are interlinked, and those IDs are reflecting that people thought they belong to together, or simply copied them. List-2014 also lists all concept IDs from Key and Comrie, and therefore also WOLD: http://clics.lingpy.org/concepts.php

We may want to add the IDS_KEY to the List-list, but for the time being, just don't make to many changes separating those, it is also not that important, if they are fuzzy. They ARE fuzzy, those lists, especially Key, but we can't change the whole world at once.

Reply:
Yes, that's why I assumed to best base the choice on Haspelmath/WOLD.

Reply:
I'll link those 3 to the same concepts, except for when the glosses are called differently in English. For example, I linked "banana tree" on Key and List to BANANA TREE but left it as BANANA on Haspelmath since there it's named "banana" and seems to refer to the fruit.